### ISSUE TITLE
Changelog for new release missing (and CHANGELOG.md outdated)

### ISSUE THREAD

**Issue by johanneskastl** [knative/func#2287]
The latest release does not have any content

```
[add release notes here]
```
https://github.com/knative/func/releases/tag/knative-v1.14.0

In addition, the CHANGELOG.md](https://github.com/knative/func/blob/main/CHANGELOG.md) is outdated.

Is there any other changelog that I missed? Having only a list of commits as changelog is not easy on the eye, but better than nothing...

**Comment by lkingland** [comment]
Hello Johannes,

Thank you for the issue.  Yes, we are aware that the changelogs and release notes are not reliably getting content.  We are improving our method of generating the release note and hope for it to be looking better beginning with the forthcoming release.

**Comment by johanneskastl** [comment]
> Thank you for the issue. The release you are linking to indeed has content. Is it possible it was not available when you created this issue?

Yes, it was only containing the placeholder I quoted. It would be very nice if the content was available as soon as the release is no longer just a tag or a pre-release. Otherwise it makes it very tedious to check for the changelog periodically, where the RSS feed alerts me on new releases...
 
> Regardless, yes, we are aware that the changelogs and release notes could be improved. We are working on our method of generating them and hope for it to be looking better beginning with the forthcoming release.

That would be very much appreciated. I am looking forward to the next release :-)

**Comment by johanneskastl** [comment]
Still relevant, do not close

**Comment by Shubham-Rasal** [comment]
Are the CHANGELOG.md and release changelog synced?
